Topics/Software_Engineering/Software_Licenses.md

```diff
@@ -0,0 +1,134 @@
+# Software Licenses
+
```

Comment:
General spelling corrections in doc:

Introduction:
- The word "framweork" should instead be "framework"

Types of Software Licenses:
- The word "multiples" should instead be "multiple"
- The word "distirbuton" should instead be "distribution"
- The word "Permissve" should instead be "Permissive"
- The word "reditribute" should instead be "redistribute"
- The word "substanctial" should instead be "substantial"
- The word "requring" should instead be "requiring"

Permissive Licenses:
- The word "approapiate" should instead be "appropriate"
- The word "propretsry" should instead be "proprietary"

Breach of Licenses:
- The word "lincenses" should instead be "licenses"
- The word "progrmas" should instead be "programs"
- The word "Genreal" should instead be "General"
- The word "redistirbutors" should instead be "redistributors"
- The word "cusotmers" should instead be "customers"
- The word "obligtions" should instead be "obligations"
- The word "exisiting" should instead be "existing"